### PR TITLE
feat: Orchestrator MultiLanguage Support

### LINKED ISSUE #5629: Orchestrator: Multi-Language Model Support
Dependencies:
- Pending Orchestrator Recognizer (SDK) work completion
- Pending size reduction from Orchestrator Core (current multilang model is 1GB)

Requirements:
- At a high level, must be able to support Multi-Lang the same way LUIS does in Composer
- Download multi-lang model and set it up correctly in orchestrator settings when user has enabled multiple languages
- Generate snapshots correctly in multiple languages

### PR DESCRIPTION
**Pull request by taicchoumsft**
## Description
Add Orchestrator Multi-language local support.  

The Orchestrator build pipeline is adjusted to use the `en` default model for English locales, and the `multilang` default model for every other locale.  

This PR does not cover warnings for unsupported languages, publishing to Azure for multilanguage bots when using Orchestrator, or a UI for the download of the models.  These items will be covered in a separate set of PRs. 

## Task Item
Closes #5629

### CODE REVIEW

File: Composer/packages/server/src/models/bot/interface.ts [hunk 1]
```diff
@@ -45,7 +45,9 @@ export interface ILuisStatusOperation {
 
 export interface IOrchestratorNLRList {
   version: string;
-  default: string;
+  defaults: {
+    [key: string]: string;
```
Comment (hatpick) [inline Composer/packages/server/src/models/bot/interface.ts:49]: prefer Record<string, string>

File: Composer/packages/server/src/models/bot/builder.ts [hunk 2]
```diff
@@ -159,19 +160,64 @@ export class Builder {
...
+    modelPath: string,
+    luFiles: FileInfo[],
+    emptyFiles: { [key: string]: boolean }
+  ) => {
+    if (!luFiles.filter((file) => !emptyFiles[file.name]).length) return;
+
     // build snapshots from LU files
-    const returnData = await this.orchestratorBuilder(luFiles, modelPath);
+    const returnData = await this.orchestratorBuilder(
+      //luFiles.filter((file) => file.name.split('.')?.[1] == 'en-us'),
```
Comment (hatpick) [inline Composer/packages/server/src/models/bot/builder.ts:217]: remove?

File: Composer/packages/server/src/models/bot/builder.ts [hunk 3]
```diff
@@ -159,19 +160,64 @@ export class Builder {
   public runOrchestratorBuild = async (luFiles: FileInfo[], emptyFiles: { [key: string]: boolean }) => {
     if (!luFiles.filter((file) => !emptyFiles[file.name]).length) return;
 
+    const [enLuFiles, multiLangLuFiles] = partition(luFiles, (f) =>
+      f.name.split('.')?.[1]?.toLocaleLowerCase().startsWith('en')
```
Comment (hatpick) [inline Composer/packages/server/src/models/bot/builder.ts:164]: nit: toLowerCase should be fine here

File: Composer/packages/server/src/models/bot/interface.ts (hunk 1 above)
Comment (taicchoumsft) [reply Composer/packages/server/src/models/bot/interface.ts]: Thanks @hatpick , resolved all your suggestions

File: Composer/packages/server/package.json [hunk 4]
```diff
@@ -70,7 +70,7 @@
     "@microsoft/bf-dispatcher": "^4.11.0-beta.20201016.393c6b2",
     "@microsoft/bf-generate-library": "^4.10.0-daily.20210129.207826",
     "@microsoft/bf-lu": "^4.12.0-dev.20210129.82760de",
-    "@microsoft/bf-orchestrator": "4.11.0-beta.20201208.165a6d6",
+    "@microsoft/bf-orchestrator": "4.11.0-beta.20210210.116dcad",
```
Comment (tsuwandy) [inline Composer/packages/server/package.json:73]: @taicchoumsft , you could use this version now 4.12.0-beta.20210218.668cdac

**Review by tsuwandy — APPROVED** [review]

File: Composer/packages/server/package.json (hunk 4 above)
Comment (taicchoumsft) [reply Composer/packages/server/package.json]: Thanks @tsuwandy , updated the package to `4.12.0-beta.20210218.668cdac`

**Review by cwhitten — DISMISSED** [review]

**Review by cwhitten — APPROVED** [review]

**Pull request merged**